File: MCP_Backend-master/src/main/java/com/greybox/mediums/models/equiweb/CIStatementRequest.java
package com.greybox.mediums.models.equiweb;

import com.greybox.mediums.models.OutletAuthRequest;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.Date;

@Data
@EqualsAndHashCode(callSuper = false)
public class CIStatementRequest extends OutletAuthRequest {
    private String accountNo;
    private String statementType;
    private String entityType;
    private Date fromDate;
    private Date toDate;
    private String transType;
}

File: micropay-mobile-apis-master/Models/CIStatementRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using micropay_apis.APIModals;

namespace micropay_apis.Models
{
	public class CIStatementRequest
	{
		public string accountNo;
		public string statementType;
        public string entityType;
        public DateTime? fromDate;
		public DateTime? toDate;
		public string transType; 
		public OutletAuthentication authRequest;
	}
}